src/main/java/core/basesyntax/strategy/mapvalidator/KeyPresentValidator.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.strategy.mapvalidator;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.Map;
+
+public interface KeyPresentValidator extends MapValidator {
+    @Override
+    default void validateMap(Map<String, Integer> fruitQuantityMap,
+                                    FruitTransaction transaction) {
+        if (!fruitQuantityMap.containsKey(transaction.getFruit())) {
+            String exceptionMessage = "Fruit from transaction = ["
+                    + transaction + "] was not found in map as it is required by "
+                    + transaction.getOperation() + " operation";
+            throw new IllegalArgumentException(exceptionMessage);
+        }
+    }
+}
```

Comment:
This interface should be in the `strategy` package according to the checklist. Please move it to the correct package.